example/src/App.js

```diff
@@ -42,18 +47,20 @@ const App = () => {
             name="CheckoutPage"
             component={CheckoutView}
             options={({ navigation }) => ({
-              title: 'Adyen React Native',
-              headerRight: () => (
-                <Button
-                  onPress={() => navigation.navigate('Settings')}
-                  title="Edit"
-                />
-              ),
+              title: 'Checkout',
+              headerRight: () => <SettingsButton navigation={navigation} />,
```

Comment:
Sonar cloud is complaining about this
https://sonarcloud.io/project/issues?resolved=false&types=CODE_SMELL&pullRequest=195&id=Adyen_adyen-react-native&open=AYe8wPYFweZLuJAQoljM

```suggestion
              headerRight: () => (<SettingsButton navigation={navigation} />),
```